ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/utils/Result.kt

```diff
@@ -0,0 +1,6 @@
+package com.revenuecat.purchases.ui.revenuecatui.utils
+
+internal sealed class Result<out A, out B> {
```

Comment:
I noticed it later, but kotlin has its own `Result` class that we can also use...Should we use that?

Reply:
Oh I thought you couldn't use it as a return type but it looks like since 1.5 you can! https://github.com/Kotlin/KEEP/blob/master/proposals/stdlib/result.md#summary

Reply:
So we can remove this now?

Reply:
Yes, I just replaced it with Kotlin's Result 😄